README.md

```diff
@@ -117,6 +117,13 @@ _More here soon!_
     be edited to make uploads optional for (all) work types, by setting
     `config.work_requires_files = false`.
 
+  * NewspaperWorks overrides Hyrax's default `:after_create_fileset` event
```

Comment:
Can we add a bit more info (very brief explanation) here? Where does the override occur? Why are we doing this override?

Reply:
Good point, will elaborate on why we override the handler in the README; will add a commit to that effect today.